src/include/settings/settings.h

```diff
@@ -22,12 +22,14 @@
 SETTING_int(port,
            "Peloton port (default: 15721)",
            15721,
+           512, 64000,
            false, false)
 
 // Maximum number of connections
 SETTING_int(max_connections,
            "Maximum number of connections (default: 64)",
            64,
+           0, 512,
```

Comment:
I'm not seeing where this is used... so maybe doesn't matter..
but min # of connections 0? Seems wrong.